### ISSUE TITLE
Port to HypothesisTests.jl?

### ISSUE THREAD

**Issue by azev77** [torfjelde/KernelGoodnessOfFit.jl#1]
Hi @torfjelde and thanks for this package!
Is there a chance you can port this to HypothesisTests.jl? 
(that way more Julia users can discover & use this)

**Comment by torfjelde** [comment]
Hey @azev77 !
So both this and the WeibullCount.jl package (ref: https://github.com/torfjelde/WeibullCount.jl/issues/1) are really not something I would suggest using :confused: 

These two packages are essentially the very first thing I did in Julia, in an attempt to learn the language. Let's just say the quality of these packages isn't something I'm particularly proud of:) So if I were to do something like that, it would have to be rewrites of the current packages.

A rewrite of the WeibullCount.jl package is very far down on my TODO-list as this is not something I need anymore, so there's only two ways this gets rewritten: a) I just really want to some random night, or b) a lot of people really want it. For now none of those have happened.

Now this package, KernelGoodnessOfFit.jl, is also something that's far down on my TODO-list but much more likely that I'll get around to at some point. Now that we have proper packages for kernel functions in Julia, etc., I've been wanting to do this for a while. But again, it's a time-issue as I have a lot of other responsibilities at the moment.

TL;DR: WeibullCount.jl is probably not going to be fixed/contributed to anything, but KernelGoodnessOfFit.jl definitively will _at some point_. Sorry :confused: 

With that being said, if you're interested in contributing to either yourself, I'd be happy to try and help where I can:)

**Comment by azev77** [comment]
Sounds good @torfjelde. 
In general whenever I see a probability distribution or hypothesis test floating around I always suggest the author contribute it to the major repo...

**Comment by torfjelde** [comment]
Good initiative:) If they didn't require a huge rewrite, this indeed would have been a reminder to do so. I'll leave this issue open though, as a reminder that I need to do this at some point!
